compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -281,6 +281,10 @@ trait PrepJSExports { this: PrepJSInterop =>
         }
       }
 
+      lazy val symOwner =
+        if (sym.isConstructor) sym.owner.owner
+        else sym.owner
```

Comment:
Consider make this a normal `val`, even if it is only used in 2/3 of the cases below. The overhead of the `lazy val` in the cases where we need it will probably outweigh the benefit of avoiding two or three point dereferences. Especially since the "Normal" case will likely be less common than the "Top-level" and "Static" ones.

In fact we should probably rename `ExportDestination.Normal` to `ExportDestination.Member`, now that it's only used for member exports.